tests/rulesets/.tflint.hcl

```diff
@@ -0,0 +1,3 @@
+plugin "azurerm" {
+    enabled = true
```

Comment:
This file is using 4 space indentation. The files above and below it use 2 spaces.